frontend/components/customizer/path-builder.tsx

```diff
@@ -153,10 +155,22 @@ export default function PathBuilder({
     })
   }
 
-  function renderLiteral(literal: string, tokenIndex: number) {
+  function renderLiteral(
+    literal: string,
+    tokenIndex: number,
+    onlyLiterals: boolean
+  ) {
     return (
-      <PathBuilderColumn key={`${tokenIndex}-${literal}`}>
-        <PathLiteral isFirstToken={tokenIndex === 0}>{literal}</PathLiteral>
+      <PathBuilderColumn
+        height={onlyLiterals ? '18px' : '78px'}
+        key={`${tokenIndex}-${literal}`}
+      >
+        <PathLiteral
+          isFirstToken={tokenIndex === 0}
+          marginTop={onlyLiterals ? '0px' : '39px'}
```

Comment:
I'd suggest passing `onlyLiterals` into `PathBuilderColumn` and `PathLiteral` and moving the conditional expression there. That way the layout components (i.e. the columns and the literals) are better encapsulated. Also it avoids duplicating the `'78px'` which should make it easier to maintain if this changes.